crates/net/nat/src/lib.rs

```diff
@@ -134,4 +201,16 @@ mod tests {
         let ip = external_ip().await;
         dbg!(ip);
     }
+
+    #[tokio::test]
+    #[ignore]
+    async fn get_external_ip_interval() {
+        reth_tracing::init_tracing();
+        let mut interval = ResolveNatInterval::interval(Default::default(), Duration::from_secs(5));
+
+        let ip = interval.tick().await;
+        dbg!(ip);
+        let ip = interval.tick().await;
+        dbg!(ip);
```

Comment:
Can we make these asserts instead of dbg?

Reply:
these tests are ignored, and only intended for debugging purposes.